### ISSUE TITLE
Table 4 results

### ISSUE THREAD

**Issue by munchmoo** [Cc-Hy/CMKD#17]
Hello,

Can I get the experimental details of results in Table 4. ? 

Because this repo has some differences from paper, it's kind of confusing.
Are these results use Resnet101 with bin num=80, just as CaDDN setting? 

Thank you.

**Comment by Cc-Hy** [comment]
Hi, thanks for your attention.
Yes, when we were working on the paper, we follow the settings from CaDDN.
And when we release the code, we replace the backbone to R50 and increase the bin number to 120, and this trade-off reduces memory consumption, speeds up training, and has similar results.

**Comment by munchmoo** [comment]
@Cc-Hy  Thank you for the reply.

Comparing the results in Table 4 and Table1, does validating on KITTI test(training: trainval set) perform better than KITTI val(training: train set) ? Because, I usually find validating on KITTI val(3769) perform better than KITTI test(7518).

(Table1)
<img width="462" alt="스크린샷 2023-02-07 오후 3 28 37" src="https://user-images.githubusercontent.com/74658678/217165550-000e0db6-d97e-4e82-a838-4d491132f06f.png">
(Table4)
<img width="294" alt="스크린샷 2023-02-07 오후 3 28 56" src="https://user-images.githubusercontent.com/74658678/217165618-5f620126-28a3-4084-a718-5ea47273e370.png">

Thank you

**Comment by Cc-Hy** [comment]
@munchmoo Hi,
There are manly the following reasons for the results in the table：
1. The settings are different. For results in table 1, i.e, test set comparisons, we follow the settings from DD3D, SOTA method of that time, and use the depth backbone pre-trained with eigen clean split for best results. See our experiment setting section and DD3D for more details. For results in table 4, we just explore the effectiveness of each component and do not use extra depth pre-training.
2. Like other dense-depth-based and BEV-based methods, CMKD performs better when the amount of training data is more sufficient. And when the training data becomes much larger, such as when we compare CMKD with other methods on Waymo dataset or Nuscenes dataset, the performance gains of CMKD are even more significant.

**Comment by ksh11023** [comment]
@Cc-Hy 
Okay I get the point.

So pre. in Table 4. indicates using COCO pretrained image backbone rather than, futher pre-trainining depth distribution module?

**Comment by xiaoxusanheyi** [comment]
> 2\. CMKD 在训练数据量更充足时表现更好。当训练数据变得更大时，例如当我们将 CMKD 与其他方法在 Waymo 数据集或 Nuscenes 数据集上进行比较时，CMKD 的性能提升更加显着。



> @Cc-Hy 感谢您的回复。
> 
> 比较表4和表1中的结果，在KITTI test(training: trainval set)上进行验证是否比KITTI val(training: train set)表现更好？因为，我经常发现在KITTI val(3769)上的验证比KITTI测试(7518)表现更好。
> 
> (表一) (表四) <img alt="스크린샷 2023-02-07 오후 3 28 37" width="462" src="https://user-images.githubusercontent.com/74658678/217165550-000e0db6-d97e-4e82-a838-4d491132f06f.png"> <img alt="스크린샷 2023-02-07 오후 3 28 56" width="294" src="https://user-images.githubusercontent.com/74658678/217165618-5f620126-28a3-4084-a718-5ea47273e370.png">
> 
> 谢谢

你好，想问一下，针对这个实验测试，你是如何进行测试的呢？，具体要修改哪里？感谢！

**Comment by xiaosenho** [comment]
如何在KITTI测试集上运行，7518个测试集样本并没有label标签值，我看到作者的test_cmdk.py都是使用验证集数据进行评估
